packages/experimental/Tabs/src/TabsItem.types.ts

```diff
@@ -14,6 +14,8 @@ export interface TabsItemTokens extends FontTokens, IBorderTokens {
    */
   indicatorColor?: string;
 
+  tabsItemOpacity?: number;
```

Comment:
Should we have platform specific tokens? This token is really only used for the Windows platform, though I can hook it up to macOS and win32 as well.

Reply:
I think we can expose it to all platforms. Maybe someone will want it on win32 one day. We can rename it to make it obvious it's windows specific, or add a comment signaling it. RN has a similar issue with its props, so, I don't think it's that weird to have some platform-ness bleed through.